setup.sh

```diff
@@ -1,10 +1,113 @@
 #!/usr/bin/env bash
 
-export SERVER_ADDRESS="$1"
-export HULY_SECRET="secret"
+HULY_VERSION="v0.6.295"
+DOCKER_NAME="huly"
+CONFIG_FILE="huly.conf"
 
-echo "Setting Huly Server Address: $SERVER_ADDRESS"
-envsubst < template.conf > nginx.conf
-envsubst < template.env > .env
+if [ -f "$CONFIG_FILE" ]; then
+    source "$CONFIG_FILE"
+fi
 
-./use-version.sh v0.6.295
+while true; do
+    if [[ -n "$HOST_ADDRESS" ]]; then
+        prompt_type="current"
+        prompt_value="${HOST_ADDRESS}"
+    else
+        prompt_type="default"
+        prompt_value="<IP>"
```

Comment:
using default value (just pressing Enter when asked for hostname/IP) produces this error:
```
./setup.sh
Enter the host address (domain name or IP) [default: <IP>]:
hostname: illegal option -- i
usage: hostname [-f] [-s | -d] [name-of-host]
```